src/ui_utils.c

```diff
@@ -305,6 +306,21 @@ static gchar *create_statusbar_statistics(GeanyDocument *doc,
 				g_string_append_printf(stats_str, "%d",
 					sci_get_style_at(doc->editor->sci, pos));
 				break;
+			case 'z':
+				zoom = sci_get_zoom(doc->editor->sci);
+				if (zoom != 0)
+				{
+					gint size, percent;
+
+					size = sci_get_fontsize(doc->editor->sci, STYLE_DEFAULT);
+					if (size > 0)
+					{
+						percent = (size + zoom) * 100 / size;
+						g_string_append_c(stats_str, ' ');
```

Comment:
this should probably use `sp` like other similar replacements above.
It also probably should be *after* the actual contents (see for `%r` and `%m`). `%Y` seems to be a little off, and given it's not targeted to the average user and always displays something it's not a good example :)

Reply:
Done.